packages/terra-spacer/docs/README.md

```diff
@@ -1,86 +0,0 @@
-# Terra Spacer
-
-This component is used to provide margin and/or padding space between two components based on the given values.
-
-## Getting Started
-
-- Install with [npmjs](https://www.npmjs.com):
-  - `npm install terra-spacer`
-
-<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
-## Peer Dependencies
-
-This component requires the following peer dependencies be installed in your app for the component to properly function.
-
-| Peer Dependency | Version |
-|-|-|
-| react | ^16.8.5 |
-| react-dom | ^16.8.5 |
-
-
-
-<!-- AUTO-GENERATED-CONTENT:END -->
-
-## Naming Convention
-
-| Spacing scale values   | Computed REM Value |
-| --------------------   | ------------------ |
-| none                   | 0                  |
-| small-2                | 0.2142857143rem    |
-| small-1                | 0.3571428571rem    |
-| small                  | 0.5rem             |
-| medium                 | 0.7142857143rem    |
-| large                  | 0.8571428571rem    |
-| large+1                | 1.0714285714rem    |
-| large+2                | 1.4285714286rem    |
-| large+3                | 2.1428571429rem    |
-| large+4                | 3.5714285714rem    |
-
-## Usage
-
-```jsx
-import React from 'react';
-import Spacer from 'terra-spacer';
-
-// Small spacing
-
-<Spacer marginTop="small" marginLeft="small-1" paddingTop="small-2">
-  <Image src="someImageSource" alt="example image" />
-<Spacer/>
-
-// Medium spacing
-
-<Spacer marginLeft="medium" marginRight="medium">
-  <Image src="someImageSource" alt="example image" />
-<Spacer/>
-
-// Large and very large spacing
-
-<Spacer paddingTop="large" paddingBottom="large+1" paddingLeft="large+2" marginLeft="large+4" marginRight="large+3">
-  <Image src="someImageSource" alt="example image" />
-<Spacer/>
-
-// Shorthand consumption
```

Comment:
Do we need to add an example demonstrating short hand vs long hand consumption?

Reply:
done here https://github.com/cerner/terra-core/pull/2872/commits/4efa60a5a6bd274e0e0926452d27d02ba78d8ad7